collectors/scm/bitbucket/src/main/java/com/capitalone/dashboard/dao/PullRequestDao.java

```diff
@@ -0,0 +1,85 @@
+package com.capitalone.dashboard.dao;
+
+import com.capitalone.dashboard.model.Commit;
+import com.capitalone.dashboard.model.CommitType;
+import com.capitalone.dashboard.model.GitRepo;
+import com.capitalone.dashboard.model.GitRequest;
+import com.capitalone.dashboard.repository.CommitRepository;
+import com.capitalone.dashboard.repository.GitRequestRepository;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+import java.util.List;
+
+@Component
+public class PullRequestDao {
```

Comment:
Per Hygieia standards, we use interface implementation model. See our current implementation of service/serviceimpl in the API layer.

Reply:
@Sbrenthughes I think the use of the interface is unnecessary here. I don't think of a use case where we want to provide multiple implementation of this class. However, if this is Hygieia standards violation then I can create an interface based implementation. Thoughts?

Reply:
We don't use the Dao naming convention. It may be better just just add this method into the class it is used.